visualizer/parseTree.js

```diff
@@ -156,6 +159,31 @@
     }
   }
 
+  // Hides or shows the semantic editor of `el`, which is a div.pexpr.
+  function toggleSemanticEditor(el) {
```

Comment:
Please add a comment that explains what this does, in a way that someone using it would understand.